controller/codebase.go

```diff
@@ -303,10 +301,6 @@ func ConvertCodebase(request *http.Request, codebase *codebase.Codebase, additio
 			Edit:    &editURL,
 		},
 	}
-	for _, add := range additional {
```

Comment:
Why remove this?

Reply:
no reason except it's never used, but I can restore it, actually.